viewPost.php

```diff
@@ -1,31 +1,41 @@
 <?php
 
 include('config/init.php');
+$comment=getAllComments($_GET['blogPostId']);
+$result=getAllCategories($_GET ['blogPostId']);
+$blogPost=getBlogPost($_GET['blogPostId']);
 
-
-$result=getAllBlogPosts();
-$blogPosts= getBlogPost($_GET['categoryId']);
-
-
-echoHeader("".$blogPosts['title'], "".$blogPosts ['description']);
+echoHeader($blogPost['title']);
 
 echo "
-<h2>".$blogPosts['title']."</h2>
-<p>".$blogPosts['content']."</p> 
+<div class='bordertop'></div>
+<h2>".$blogPost['title']."</h2>
+<h5> By ".$blogPost['authorName']."</h5>
+<p>".$blogPost['content']."</p> 
 ";
 
 if(isset($_REQUEST['newComment'])){
```

Comment:
You'll want to process all forms at the top of the page so that you can redirect after processing it. If you need to echo something like "invalid comment"...I bet you can figure out how to do that